src/dotnet/Core.Server/Blobs/ContentSaver.cs

```diff
@@ -19,4 +19,13 @@ public async Task Remove(string contentId, CancellationToken cancellationToken)
         await _blobStorage.Delete(contentId, cancellationToken).ConfigureAwait(false);
         await Task.Delay(PostOperationDelay, CancellationToken.None).ConfigureAwait(false);
     }
+
+    public async Task<bool> Exists(string contentId, CancellationToken cancellationToken)
+        => await _blobStorage.Exists(contentId, cancellationToken).ConfigureAwait(false);
```

Comment:
omit await

Reply:
why?

Reply:
Usually we try to return the task directly without awaiting.